.changeset/config.json

```
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
+  "changelog": ["@changesets/changelog-github", { "repo": "refinedev/refine" }],
```

is this `"repo": "refinedev/refine"` a typo? 

```suggestion
  "changelog": ["@changesets/changelog-github", { "repo": "refinedev/refine" }],
```

---

yes, its typo. Thanks for catching it.

---

fixed.